src/docker-osx-dev

```diff
@@ -970,6 +979,8 @@ function watch_and_sync {
 #
 function install {
   log_info "Starting install of docker-osx-dev"
+  cp $SCRIPT /usr/local/bin/docker-osx-dev
```

Comment:
This seems to be a change unrelated to this PR. Could you submit a separate PR for it so we the commit and discussion around it can happen in isolation?